README.md

```diff
@@ -1058,9 +1059,10 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 * [CheatSheet](https://www.mediaatelier.com/CheatSheet/) -  Hold the ⌘-Key to get a list of all active shortcuts of the current application. It's as simple as that. ![Freeware][Freeware Icon]
 * [DisplayBuddy](https://displaybuddy.app) - Control the brightness, contrast, input source and more of your external display directly from your Mac.
 * [f.lux](https://justgetflux.com/) - Makes the color of your computer's display adapt to the time of day. ![Freeware][Freeware Icon]
-* [Lunar](https://lunar.fyi/) -  Help you adujst brightness, contrast and volumn of your external display. [![Open-Source Software][OSS Icon]](https://github.com/alin23/Lunar) ![Freeware][Freeware Icon]
 * [Grayscale Mode](https://github.com/rkbhochalya/grayscale-mode) - An open source macOS app that lets you quickly toggle grayscale filter right from your menu bar or using a keyboard shortcut (⌥⌘G). [![Open-Source Software][OSS Icon]](https://github.com/rkbhochalya/grayscale-mode) ![Freeware][Freeware Icon]
+* [IconChanger](https://github.com/underthestars-zhy/IconChanger) - Simply change your app's icon on macOS. Just a click. [![Open-Source Software][OSS Icon]](https://github.com/underthestars-zhy/IconChanger) ![Freeware][Freeware Icon]
```

Comment:
@Lin-Youth Nice tool, it doesn't appear to be a desktop application.

Reply:
I put a screenshot of the desktop app running on my mac below: 
<img width="750" alt="image" src="https://github.com/jaywcjlove/awesome-mac/assets/26051323/a53a8d23-fd10-40b6-9149-86485fce2968">

Reply:
I can't find the app download link for it

@Lin-Youth

Reply:
Just at the release page：https://github.com/underthestars-zhy/IconChanger/releases

Reply:
you can remove it, thanks

Reply:
done. Only Jellyfin added.